apps/site/lib/site_web/controllers/customer_support_controller.ex

```diff
@@ -301,7 +301,7 @@ defmodule SiteWeb.CustomerSupportController do
           parsed_dt
       end
 
-    local_parsed_date_time = Util.to_local_time(parsed_date_time)
+    local_parsed_date_time = Timex.to_datetime(parsed_date_time, "America/New_York")
```

Comment:
Is this different from what `Util.to_local_time` is doing?

Reply:
Yes, I think the previous conversion using `Util` isn't what we want here.
For example today at 1 pm should be:
`DateTime<2020-11-23 13:00:00-05:00 EST America/New_York>`, but `Util.to_local_time` returns:
`DateTime<2020-11-23 08:00:00-05:00 EST America/New_York>`

Reply:
Got it. `Util.to_local_time` is probably doing too much (some of it could live in `parse`?) so avoiding it is a reasonable choice.